docs/coreneuron/compatibility.rst

```diff
@@ -0,0 +1,133 @@
+CoreNEURON compatibility
+########################
+CoreNEURON is designed as a library within the NEURON simulator and can transparently handle all spiking network simulations, including gap junction coupling, with the **fixed time step method**.
+In order to run a NEURON model with CoreNEURON certain conditions must be met:
+
+* MOD files must be ``THREADSAFE`` (`more information <https://neuron.yale.edu/neuron/docs/multithread-parallelization>`_)
```

Comment:
I think that for this it is more useful for users to provide some practical instructions. I think it's worth creating a new section for that so I'll add this together with a few words

Reply:
Great! Feel free to push to this PR.